Task2/Task2/Task2/Services/ServicesByProduct.cs

```diff
@@ -0,0 +1,146 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Task2.Exceptions;
+using Task2.Products;
+
+namespace Task2.Services
+{
+    public static class ServicesByProduct
+    {
+        public static Bolt AdditionOfTwoProducts(Bolt leftProduct, Bolt rightProduct)
```

Comment:
The same issue about arguments null check.

Reply:
fixed